index.html

```diff
@@ -148,8 +148,8 @@ <h3 class="head-three desktop">FACEBOOK</h3>
                 <div class="card flex">
```

Comment:
- [ ] Kindly rectify the color contrast issue for the `language/technology` badge with the project cards.
![language-badges-color-access](https://user-images.githubusercontent.com/44057863/167713229-25330679-c221-4bca-83e6-f5226ddfda88.png)